orchestra/tests/unit/specs/mistral/test_workflow_spec_validate.py

```diff
@@ -115,3 +115,61 @@ def test_empty_task_default_list(self):
         wf_spec = self.instantiate(wf_def)
 
         self.assertDictEqual(wf_spec.inspect(), {})
+
+    def test_type_direct(self):
```

Comment:
Can you rename this function to test_workflow_type_direct so looking at the test name we know it's testing workflow type and not "other" types?